java/client/src/main/java/glide/api/commands/GenericClusterCommands.java

```diff
@@ -23,27 +26,29 @@ public interface ClusterBaseCommands {
      *     <p><code>
      * Object result = client.customCommand(new String[]{ "CLIENT", "LIST", "TYPE", "PUBSUB" }).get();
      * </code>
-     * @param args Arguments for the custom command including the command name
-     * @return A <em>CompletableFuture</em> with response result from Redis
+     * @param args Arguments for the custom command including the command name.
+     * @return Response from Redis containing an <code>Object</code>.
```

Comment:
"Response" -> "A response"
This is both consistent with the return comment earlier in the PR ("A Future to connect and return a RedisClient"), and conveys the fact that response is not an object type.

minor: Either way, docs should be consistent - either all return comments start with "A bla bla", or just "bla bla".

Reply:
I will update the connection response (above) in another PR.  This wording has been vetted several times, and I think it's being used already for command calls everywhere.  I will raise a PR to discuss this, and if you'd prefer "A response" instead, we can change it.  
However, we wrap all object types with a link or `<code>` tags, and since "Response" doesn't have one, it's not an object.